autoware_launch/launch/autoware.launch.xml

```diff
@@ -92,6 +92,12 @@
     </include>
   </group>
 
+  <!-- API -->
+  <include file="$(find-pkg-share tier4_autoware_api_launch)/launch/autoware_api.launch.xml">
+    <arg name="init_simulator_pose" value="true"/>
+    <arg name="init_localization_pose" value="false"/>
+  </include>
+
```

Comment:
https://github.com/tier4/autoware_launch/blob/d629633e3c9533fb54e715eca29a6c0aded56661/autoware_launch/launch/autoware.launch.xml#L92-L98

Reply:
Flipped?
![image](https://user-images.githubusercontent.com/31987104/162562036-fa23212f-2d00-4404-8c40-9dd9bab3d15d.png)

Reply:
Thank you, I have fixed the arguments.